src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -825,7 +825,8 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
 
         const myCallMemberEvent = roomState.getStateEvents(EventType.GroupCallMemberPrefix, localUserId) ?? undefined;
         const content = myCallMemberEvent?.getContent() ?? {};
-        const legacy = "memberships" in content || this.useLegacyMemberEvents;
+        const legacy =
+            (("memberships" in content && content.membership?.length) ?? 0 > 0) || this.useLegacyMemberEvents;
```

Comment:
Is the second `membership` meant to be `memberships` of a `*.call.member` event?
```suggestion
            (("memberships" in content && content.memberships?.length) ?? 0 > 0) || this.useLegacyMemberEvents;
```

Reply:
of course!! thx